spec/index.html

```diff
@@ -2470,106 +2413,106 @@ <h3>Grammar summary</h3>
     <a href="#propertyEltList">propertyEltList</a><br />
     end-element()</td>
     </tr>
     <tr>
-    <td><a href="#emptyPropertyElt"></a></td> <td>start-element(<a href="#eventterm-element-URI">URI</a> == <a href="#propertyElementURIs">propertyElementURIs</a> ),
+    <td><a href="#emptyPropertyElt"></a></td> <td>start-element(<a href="#eventterm-element-URI">IRI</a> == <a href="#propertyElementURIs">propertyElementURIs</a> ),
 
     <a href="#eventterm-element-attributes">attributes</a> == set(<a href="#idAttr">idAttr</a>?, ( <a href="#resourceAttr">resourceAttr</a> | <a href="#nodeIdAttr">nodeIdAttr</a> | <a href="#datatypeAttr">datatypeAttr</a> )?, <a href="#propertyAttr">propertyAttr</a>*))<br />
     end-element()</td>
     </tr>
 
     <tr>
-    <td><a href="#idAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">URI</a> == <code>rdf:ID</code>,
+    <td><a href="#idAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">IRI</a> == <code>rdf:ID</code>,
     <a href="#eventterm-attribute-string-value">string-value</a> == <a href="#rdf-id">rdf-id</a>)</td>
 
     </tr>
     <tr>
-    <td><a href="#nodeIdAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">URI</a> == <code>rdf:nodeID</code>,
+    <td><a href="#nodeIdAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">IRI</a> == <code>rdf:nodeID</code>,
     <a href="#eventterm-attribute-string-value">string-value</a> == <a href="#rdf-id">rdf-id</a>)</td>
 
     </tr>
     <tr>
-    <td><a href="#aboutAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">URI</a> == <code>rdf:about</code>,
-    <a href="#eventterm-attribute-string-value">string-value</a> == <a href="#URI-reference">URI-reference</a>)</td>
+    <td><a href="#aboutAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">IRI</a> == <code>rdf:about</code>,
+    <a href="#eventterm-attribute-string-value">string-value</a> == <a href="#URI-reference">IRI</a>)</td>
 
     </tr>
     <tr>
-    <td><a href="#propertyAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">URI</a> == <a href="#propertyAttributeURIs">propertyAttributeURIs</a>,
+    <td><a href="#propertyAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">IRI</a> == <a href="#propertyAttributeURIs">propertyAttributeIRIs</a>,
     <a href="#eventterm-attribute-string-value">string-value</a> == <a>anyString</a>)</td>
 
     </tr>
     <tr>
-    <td><a href="#resourceAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">URI</a> == <code>rdf:resource</code>,
-    <a href="#eventterm-attribute-string-value">string-value</a> == <a href="#URI-reference">URI-reference</a>)</td>
+    <td><a href="#resourceAttr"></a></td> <td>attribute(<a href="#eventterm-attribute-URI">IRI</a> == <code>rdf:resource</code>,
+    <a href="#eventterm-attribute-string-value">string-value</a> == <a href="#URI-reference">IRI</a>)</td>
```

Comment:
This says "string-value == IRI" but IRI is not a string any more.

Should it be setting the "iri" attribute, not "string-value"?

Reply:
On re-inspection, the current definition of `IRI` (was `URI`) is related to the expanded attribute name, not the attribute value. The previous `URI-string-value` was used to construct a URI/IRI from the normalized value. To disambiguate, I'll change those places where `IRI` is used as the event value to `rdf-term`, which is used when the value can be either a Blank Node or an IRI.

Reply:
Please check again; I've created an `rdf-term` accessor, where there was ambiguity with the name of the event vs. the value. I also defined a `string` function to get the string value of an IRI.